### DISCUSSION TITLE
... new Builder(nuxt) equivalent ?

### DISCUSSION THREAD

**Discussion by JohnMica** [nuxt/nuxt#18213] [Questions]
hi there,

in Nuxt 2 we could do something like this when using `express` to add middleware 

```
const { Nuxt, Builder } = require('nuxt')
...
 if (nuxtConfig.dev) {
    const builder = new Builder(nuxt)
    await builder.build()
  }
```
since  neither `Nuxt` nor `Builder` are exported in Nuxt, not sure how to deal with this in Nuxt 3

any pointers to where this might be in the new version, or even if its neccessary anymore ?

**Comment by wobsoriano** [comment]
Related https://github.com/nuxt/framework/discussions/1451

**Comment by JohnMica** [comment]
thanks @wobsoriano for the quick reply
